src/test/java/ParserTest.java

```diff
@@ -0,0 +1,55 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import chatter.ChatterException;
+import chatter.Parser;
+import chatter.command.AddCommand;
+import chatter.command.Command;
+import chatter.command.DeleteCommand;
+import chatter.command.ExitCommand;
+import chatter.command.ListCommand;
+
+public class ParserTest {
+    @Test
+    public void testParseExit() throws ChatterException {
```

Comment:
perhaps function names can be improved by referring to the style guide! eg: testFunction_argument_expectedOutput